src/common/Common.h

```diff
@@ -69,6 +69,13 @@ enum TimeConstants
     IN_MILLISECONDS = 1000
 };
 
+enum TimeFormats
```

Comment:
This enum belongs together with the function inside Util.h file, do not put it here and #include "Common.h" in Util.h (more includes is bad)